packages/pwa-kit-dev/src/utils/script-utils.test.js

```diff
@@ -276,6 +358,7 @@ describe('scriptUtils', () => {
             'should push a built bundle and handle status codes (%p)',
             async ({projectSlug, targetSlug, expectedURL, status}) => {
                 readJson.mockReturnValue(pkg)
+                execSync.mockReturnValue(JSON.stringify(dependencyTreeMockData.noPwaKitPackages))
```

Comment:
mock the execSync method to avoid timing out this test because of running `npm ls --all`